com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -504,33 +504,21 @@ internal void InvokeBehaviourOnGainedOwnership()
             }
         }
 
-        internal void ResetNetworkStartInvoked()
+        internal void InvokeBehaviourNetworkSpawn(Stream stream)
         {
-            if (ChildNetworkBehaviours != null)
```

Comment:
don't we need this at all?
(potential NullReferenceException on ChildNetworkBehaviours.Count call?)

Reply:
No this is non relevant legacy code. `ChildNetworkBehaviours` is a property which self initializes in the getter, so this can never be null. I'll remove this check.